posts/2021-09-09-Rust-1.55.0.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: "Announcing Rust 1.55.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.55.0. Rust is a programming language empowering everyone
+to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.55.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.55.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-55-2021-09-09
+
+## What's in 1.55.0 stable
+
+### Cargo deduplicates compiler errors
+
+In past releases, when running `cargo test`, `cargo check --all-targets`, or similar commands which built the same Rust crate in multiple configurations, errors and warnings could show up duplicated as the rustc's were run in parallel and both showed the same warning.
+
+For example, in 1.54.0, output like this was common:
+
+```
+$ cargo +1.54.0 check --all-targets
+    Checking foo v0.1.0
+warning: function is never used: `foo`
+ --> src/lib.rs:9:4
+  |
+9 | fn foo() {}
+  |    ^^^
+  |
+  = note: `#[warn(dead_code)]` on by default
+
+warning: 1 warning emitted
+
+warning: function is never used: `foo`
+ --> src/lib.rs:9:4
+  |
+9 | fn foo() {}
+  |    ^^^
+  |
+  = note: `#[warn(dead_code)]` on by default
+
+warning: 1 warning emitted
+
+    Finished dev [unoptimized + debuginfo] target(s) in 0.10s
+```
+
+In 1.55, this behavior has been adjusted to deduplicate and print a report at the end of compilation:
+
+```
+$ cargo +1.55.0 check --all-targets
+    Checking foo v0.1.0
+warning: function is never used: `foo`
+ --> src/lib.rs:9:4
+  |
+9 | fn foo() {}
+  |    ^^^
+  |
+  = note: `#[warn(dead_code)]` on by default
+
+warning: `foo` (lib) generated 1 warning
+warning: `foo` (lib test) generated 1 warning (1 duplicate)
+    Finished dev [unoptimized + debuginfo] target(s) in 0.84s
+```
+
+### Faster, more correct float parsing
+
+The standard library's implementation of float parsing has been updated to use the Eisel-Lemire algorithm, which brings both speed improvements and improved correctness. In the past, certain edge cases failed to parse, and this has now been fixed.
+
+Read more details on the new implementation [here](https://github.com/rust-lang/rust/pull/86761).
```

Comment:
Re-worded it slightly to be more of a call to action, and replaced "here" with a description for the link, as "here" is generally considered bad style for links.
```suggestion
You can read more details on the new implementation [in the pull request description.](https://github.com/rust-lang/rust/pull/86761).
```